### ISSUE TITLE
Move mail modules to unused

### ISSUE THREAD

**Issue by mtissington** [webmin/webmin#1885]
I do NOT have sendmail (or other installed) on debian 11.

How do I move Fetchmail, Procmail and Read User mail to the list of unused modules?
(yes I know I can delete them but not what I'm asking)

thanks

**Comment by chris001** [comment]
"Webmin also has an `Un-used Modules` category, which will contain **modules for software not installed on a given system**. "
http://doxfer.webmin.com/Webmin/Webmin_Modules

**Comment by mtissington** [comment]
Yes, but I see no way to move a module into that category.  Reassign Modules does not give me that option.

**Comment by chris001** [comment]
You'd have to uninstall the package that is associated with the module.  So to move `fetchmail` into `Un-used Modules`, you'd uninstall it with the command `sudo apt remove --auto-remove fetchmail` on debian type distributions.

**Comment by mtissington** [comment]
fetchmail and procmail are NOT installed - I have even tried to remove them to confirm.
(if it makes a difference, this is on debian 11)

**Comment by jcameron** [comment]
You can force Webmin to re-detect which modules are unused via the refresh modules button on the left menu..

**Comment by mtissington** [comment]
Yes, tried that too - it seems to be falsely detecting mail modules (which have NEVER been installed).

**Comment by jcameron** [comment]
What happens if you open the Sendmail module - does it show any error message about Sendmail not being installed.

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run.

⚠️ If the issue is resolved, please take a moment to close it to help keep the project organized.

**Comment by mtissington** [comment]
Fetchmail is now in the unused category. However Read Mail is still in Servers and can't be reassigned to unused.

**Comment by iliaross** [comment]
> Fetchmail is now in the unused category. However Read Mail is still in Servers and can't be reassigned to unused.

Is `fetchmail` command installed?

**Comment by mtissington** [comment]
Seems like it's not installed on Debian 13 now.

**Comment by iliaross** [comment]
So, install it, then click "Refresh Modules" in the navigation menu and see if the problem is solved.

**Comment by mtissington** [comment]
OK, the problem with fetchmail seems to be resolved.

1) Install fetchmail, refresh modules, Servers/fetchmail present
2) Uninstall fetchmail, refresh modules, fetchmail is back in unused.

**Comment by iliaross** [comment]
Good to hear that! Thanks for the feedback! I’ll close this issue now.
